avaframe/runScripts/runAna5Hybrid.py

```diff
@@ -0,0 +1,146 @@
+"""
+Run a combination of the DFA kernel to get an alphaBeta path
+to run alphaBeta model to get the alpha angle
+and run the DFA kernel again
+"""
+
```

Comment:
This is basically the core function / algorithm (!) of ana5 so it should move there.
Or even more: should this be a com - module instead of an ana module? (up for discussions)